compiler-rt/test/ubsan/TestCases/Misc/Linux/sigaction.cpp

```diff
@@ -7,6 +7,9 @@
 // Reason unknown, needs debugging.
 // XFAIL: target=aarch64{{.*}} && ubsan-tsan
 
+// This test also fail with x86-64 and bfd, so disable it for now
+// UNSUPPORTED: ubsan-tsan
+
```

Comment:
This doesn't seem to be related to the SYCL project, so the fix should go directly to llvm.org.